src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -520,12 +522,12 @@ public int Main(string[] args)
 
         private Reports CreateReports(bool verbose, bool quiet)
         {
-            IReport output = new Report(AnsiConsole.Output);
+            IReport output = new Report(AnsiConsole.GetOutput(_runtimeEnv));
             var reports = new Reports()
             {
                 Information = output,
                 Verbose = verbose ? output : new NullReport(),
-                Error = new Report(AnsiConsole.Output),
+                Error = new Report(AnsiConsole.GetError(_runtimeEnv)),
```

Comment:
nit: move `Quiet` from the line below to the initializer and then you would be able to return `reports` directly (RVO :smile:)